src/contributors/SwimLaneContributor.ts

```diff
@@ -190,7 +190,7 @@ export class SwimLaneContributor extends Contributor {
             columnStats: new Map<number, LaneStats>(),
             globalStats: {
                 min: Number.MAX_VALUE,
-                max: Number.MIN_VALUE,
+                max: -Number.MAX_VALUE,
```

Comment:
Could you add some coments to explain min variable is positive and why max value is negative ? Thks